src/js/views/patients/actions/actions.scss

```diff
@@ -16,3 +16,16 @@
 .actions__due .is-overdue {
   color: color(red);
 }
+
+.actions__due {
+  border-radius: 4px 0 0 4px;
+}
+
+.actions__time {
+  border-left: 1px solid white;
```

Comment:
this is kind of a lazy hack, and is noticeable if the screen gets big enough. Done so the button doesn't visibly shift on hover, when the left border goes from 0px to 1px

Reply:
why is it visible on a big screen?  What if the color was `transparent`?

Reply:
because there's 1px on the top and bottom of the border that "isn't there." And it'll need to be transparent anyway (if this is the way to go), because of the highlighting on the row if it's selected (click on the image and zoom way in)
![Screen Shot 2020-01-28 at 9 33 57 AM](https://user-images.githubusercontent.com/8376505/73278710-9b600700-41b1-11ea-9968-65ace84d7032.png)